docs/best-practices/project-launch-checklist.md

```diff
@@ -1,15 +1,22 @@
-# Open Source Project Launch Checklist
+# Project Launch Checklist
 
 Gefeliciteerd! Je leest dit omdat je van plan bent om je project te open sourcen!
-Deze checklist is gecreëerd om je te helpen om je eigen open source projecten te starten waar dit strategische relevant is.
-Hiermee nemen we je mee in de zaken waar je rekening mee moet houden voor dat je besluit een open source project te starten.
+
+Deze checklist is gecreëerd om je te helpen om je eigen open source projecten te starten waar dit
+strategische relevant is. Hiermee nemen we je mee in de zaken waar je rekening mee moet houden voor
+dat je besluit een open source project te starten.
 
 Vul voor jezelf deze checklist in en neem contact op met de OSPO om het te bespreken.
 
-Deze checklist is bedoeld voor de grotere softwareprojecten.
-Voor kleine projecten van enkele honderden regels code - zoals een klein scriptjes - is een aparte checklist opgesteld met minder criteria.
+Deze checklist is bedoeld voor de grotere softwareprojecten. Voor kleine projecten van enkele
+honderden regels code - zoals een klein scriptjes - is een aparte checklist opgesteld met minder
+criteria.
```

Comment:
```suggestion
Deze checklist is bedoeld voor de grotere softwareprojecten. Voor kleine projecten van enkele
honderden regels code - zoals een klein scriptjes - voldoet een beperkte lijst van criteria.
```
Aanpassing omdat deze lijst (nog) niet beschikbaar is. Staat los van de wijzigingen van dit PR, maar misschien wel goed om mee te nemen.